ui/src/app/kvm/components/KVMHeaderForms/ComposeForm/ComposeFormFields/ComposeFormFields.test.tsx

```diff
@@ -342,7 +342,7 @@ describe("ComposeFormFields", () => {
     expect(wrapper.find("FormikField[name='cores']").exists()).toBe(false);
     expect(wrapper.find("FormikField[name='pinnedCores']").exists()).toBe(true);
     expect(wrapper.find("FormikField[name='pinnedCores']").prop("help")).toBe(
-      "2 cores available (free indices: 0-1)"
+      "2 cores available (unpinned indices: 0-1)"
```

Comment:
I've changed the language from `free` to `unpinned` since I think `free` implies that those that aren't listed cannot be pinned to even though they can.